features/placeholder.yml

```diff
@@ -0,0 +1,7 @@
+name: ::placeholder
+description: The `::placeholder` CSS pseudo-element selects text in `<input>` and `<textarea>` elements when no value is set.
+spec: https://drafts.csswg.org/css-pseudo-4/#placeholder-pseudo
+caniuse: css-placeholder
+group: selectors
```

Comment:
I wondered whether this should live with the placeholder attribute. I think they're distinct because they have dramatically different support histories. That said:

```suggestion
group: selectors
# TODO: create some kind of "input placeholders" feature when https://github.com/web-platform-dx/web-features/issues/971 is resolved, combining placeholder selectors and the element attr
```